content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md

```diff
@@ -16,6 +16,10 @@ TLS origination for traffic to external services.
 
 *   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
 
+    {{< tip >}}
+    If you used Helm with default values to install Istio, then the egress gateway is disabled by default. Please enable it and reinstall Istio before continuing.
```

Comment:
Where can a reader find the instructions to perform those tasks?

Reply:
Fixed. Thanks.